est/001.est

```diff
@@ -0,0 +1,20 @@
+date: 28-02-2016
+author: Dali Freire
+method: champions.pert
+scope:
+  1: Infrastructure setup
```

Comment:
@original-brownbear I understand your viewpoint, and agree with you. However, I need some help here because I don't know the activities involved in the 'Infrastructure setup'. I just think that something must be done, but I dont know exactly what (I just read the readme and some docs from Rultor project, but I never contributes for him). Maybe I just remove this item for now... what do you think?